extensions/wikia/AdEngine/js/spec/video/uiTemplate.spec.js

```diff
@@ -0,0 +1,73 @@
+/*global describe, it, expect, modules*/
+describe('ext.wikia.adEngine.video.player.uiTemplate', function () {
+	'use strict';
+
+	var UI = {
+			CLOSE_BUTTON: 'closeButton',
+			REPLAY_OVERLAY: 'replayOverlay'
+		},
+		noop = function () {},
+		mocks = {
+			videoSettings: {
+				isAutoPlay: noop,
+				isSplitLayout: noop
+			}
+		};
+
+	function getModule() {
+		return modules['ext.wikia.adEngine.video.player.uiTemplate']();
+	}
+
+	beforeEach(function () {
+		spyOn(mocks.videoSettings, 'isAutoPlay');
+		mocks.videoSettings.isAutoPlay.and.returnValue(true);
+
+		spyOn(mocks.videoSettings, 'isSplitLayout');
+		mocks.videoSettings.isSplitLayout.and.returnValue(false);
+	});
+
+	it('Should should show close button element if there is no autoplay and there is no split screen', function () {
+		var UITemplate = getModule(),
+			videoSettings = mocks.videoSettings;
+
+		mocks.videoSettings.isAutoPlay.and.returnValue(false);
+		mocks.videoSettings.isSplitLayout.and.returnValue(false);
+
+		expect(UITemplate.selectTemplate(videoSettings)).toContain(UI.CLOSE_BUTTON);
```

Comment:
What do you think about storing result: `UITemplate.selectTemplate(videoSettings)` ?
Then you would have on `actual` instead of two.